pkgs/development/python-modules/requests-futures/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, requests, lib, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "requests-futures";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "35547502bf1958044716a03a2f47092a89efe8f9789ab0c4c528d9c9c30bc148";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
```

Add reason why tests are disabled. We can at least try that the module imports correctly:

```suggestion
  # tests are disabled because they require being online
  doCheck = false;

  pythonImportsCheck = [ "requests_futures" ];
```